adcc/ExcitedStates.py

```diff
@@ -586,3 +654,60 @@ def describe_amplitudes(self, tolerance=0.01, index_format=None):
                 ret += "\n"
                 ret += separator
         return ret[:-1]
+
+    @property
+    def excitations(self):
+        """
+        Provides a list of Excitations, i.e., a view to all individual
+        excited states and their properties. Still under heavy development.
+        """
+        class Excitation:
+            pass
+
+        excitations = []
+        for i in range(self.size):
+            e = Excitation()
+            e.idx = i
```

Comment:
`e.index`. The two latters do not hurt. Also I would have a `kind` and maybe a `method` attribute.

Reply:
`kind` is not always set, so I left it out. Otherwise done.